client/analytics/report/index.js

```diff
@@ -2,35 +2,91 @@
 /**
  * External dependencies
  */
-import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { applyFilters } from '@wordpress/hooks';
+import { Component, Fragment } from '@wordpress/element';
 import PropTypes from 'prop-types';
+import { find } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import ExampleReport from './example';
-import RevenueReport from './revenue';
+import Header from 'layout/header';
+import OrdersReport from './orders';
 import ProductsReport from './products';
-import OrdersReport from './orders/';
+import RevenueReport from './revenue';
+import useFilters from 'components/higher-order/use-filters';
+
+const REPORTS_FILTER = 'woocommerce-reports-list';
```

Comment:
Thinking ahead, would it make sense to create a `fiters.js` where we could define these constants? Having them in one place would help with discoverability -- and also if we have areas where we use the same filter in multiple files.

Reply:
I was thinking of the action type / constant pattern when I wrote that comment, and think we could document them in the constant file. 

I don't feel strongly either way though, so I'd be fine with your other suggestion as well. Guess we should get someone else's opinion?